### PR TITLE
Add NLC class descriptions and correct dependencies

### LINKED ISSUE #2: Add descriptions to NLC classifications
Descriptions will be used when prompting the user to choose an action (medium confidence).

Sample NLC.json

```
"classes": [
        {
            "class": "weather",
            "description": "Display weather forecast",
            "emittarget": "weather.js",
            "texts": [
                "What is the weather?",
              ...
```

### PR DESCRIPTION
**Pull request by jlpadilla**
resolves #2

### CODE REVIEW

File: src/nlc/NLC.json [hunk 1]
```diff
@@ -4,23 +4,24 @@
...
             ]
         },
         {
             "class": "bluemix.space.set",
+            "description": "Set the current Bluemix space",
             "emittarget": "bluemix.space.set",
             "texts": [
                 "change space",
                 "use space",
                 "swith to space",
```
Comment (aeweidne) [inline src/nlc/NLC.json]: typo here, not yours though.

**Comment by aeweidne** [comment]
+1 other than the typo, up to you if you want to fix it

**Comment by jlpadilla** [comment]
@aeweidne Typo fixed. Thanks!

**Comment by aeweidne** [comment]
re-running dependency-ci

**Comment by aeweidne** [comment]
gonna merge despite failing dependency ci, the service appears to be having issues

**Pull request merged**